### PR TITLE
Check supplied location within fence rather than current loc

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

File: libraries/AP_Common/Location.h [hunk 1]
```diff
@@ -58,7 +58,8 @@ class Location_Class : public Location
     // get position as a vector from home (x,y only or x,y,z)
```
Comment (OXINARF) [inline libraries/AP_Common/Location.h]: Do you want to change this comment or do you want me to do it on top?

**Comment by OXINARF** [comment]
@peterbarker Can you please rebase on master so CI can pass?

**Comment by peterbarker** [comment]
On Thu, 14 Dec 2017, Francisco Ferreira wrote:

> Do you want to change this comment or do you want me to do it on top?

Fixed it in one of the existing commits.

**Comment by peterbarker** [comment]
On Thu, 14 Dec 2017, Francisco Ferreira wrote:

> @peterbarker Can you please rebase on master so CI can pass?

Rebased and pushed up.

**Comment by OXINARF** [comment]
Merged, thanks!

**Pull request merged**